packages/checkbox/checkbox.scss

```diff
@@ -73,6 +77,20 @@ $checkbox-focus-outline-color: $himmel;
     }
 }
 
+.jkl-checkbox-group {
+    border: 0;
+    padding: 0;
+    margin: 0;
+    line-height: 1;
+
+    & > legend {
+        font-size: $font-size-2;
+        line-height: $line-height-2;
```

Comment:
Should we use one of the mixins for this? Does it have the same style as some other text?

Reply:
These will probably have to be changed anyways (see #330)